docs/processing.md

```diff
@@ -147,49 +147,54 @@ the design phase. The Python-like language currently assumes statements can only
 
 Remove a forbidden attribute such as `http.request.header.authorization` from spans only
 
-`delete(attributes["http.request.header.authorization"]) from traces`
-`delete(attributes["http.request.header.authorization"])`
+`delete(attributes["http.request.header.authorization"]) from traces`  
+`delete(attributes["http.request.header.authorization"])`  
```

Comment:
Why do we double all the lines w/o a "from" statement?

Reply:
According to the [example description](https://github.com/open-telemetry/opentelemetry-collector/blob/ab796818282d61ba0bec0e372adb126f73c169a2/docs/processing.md?plain=1#L145) its showing 2 different types of languages

Reply:
I don't understand sorry. Do you want to say that the only diff is "from signal" everywhere?

Reply:
Some of the example show the difference better

```
Filtering out signals such as by removing all metrics with a `http.target` of `/health`

`drop() where attributes["http.target"] = "/health" from metrics`
`if attributes["http.target"] = "/health": drop()`

Attach information from resource into telemetry, for example adding certain resource fields as metric attributes

`set(attributes["k8s_pod"], resource.attributes["k8s.pod.name"]) from metrics`
`attributes["k8s_pod"] = resource.attributes["k8s.pod.name"]`
```

But ya, for the scenarios where the work is all done in a function its basically the same

Reply:
Ya, but the second line does not have a "signal" part, which means that the "python" code applies to all not only to metrics.